assets/src/edit-story/elements/video/controls.js

```diff
@@ -60,6 +61,7 @@ const ButtonWrapper = styled.div.attrs({ role: 'button', tabIndex: -1 })`
   pointer-events: initial;
   width: ${PLAY_BUTTON_SIZE}px;
   height: ${PLAY_BUTTON_SIZE}px;
+  overflow: hidden;
```

Comment:
Right now it doesn't show scrollbars 👍🏻 , but the drop shadow is also cut, do we want to fix it too? (either enlarge the element and add padding for shadow or make changes in the parent?)
![image](https://user-images.githubusercontent.com/21036224/119476220-26bdbe00-bd4e-11eb-97f2-3f176074b313.png)

Reply:
Adjusted:
![Screenshot 2021-05-25 at 14 05 33](https://user-images.githubusercontent.com/3294597/119487616-e95f2d80-bd59-11eb-81b6-e84bbd1a29bd.png)

Reply:
Hm, the shadow area shouldn't really cover the handle visibly though, I think. Will move it for correctness, thanks for the detailed check!

Reply:
Done!

Reply:
> Hm, the shadow area shouldn't really cover the handle visibly though, I think. Will move it for correctness, thanks for the detailed check!

For me visually it was fine, but it was covering that little part of the handle - making it unactionable (and would trigger play/pause). All great now.